src/test/resources/fields/655/marc_655_with_multiple_subfield_a.jsonl

```diff
@@ -10,26 +10,14 @@
           {
```

Comment:
can we rename this file? File name says "multiple_subfield_a". However, there is only one subfield.